test/blackbox/api/fastrtps_deprecated/PubSubReader.hpp

```diff
@@ -289,7 +289,7 @@ class PubSubReader
         return *subscriber_;
     }
 
-    void init()
+    void init(bool must_fail = false)
```

Comment:
```suggestion
    void init()
```

It is not really being used. It can be removed. In fact is issuing the unused parameter warning.